app/web/components/LocationAutocomplete/LocationAutocompleteOutlined.tsx

```diff
@@ -0,0 +1,175 @@
+import {
+  Autocomplete,
+  AutocompleteChangeReason,
+  InputAdornment,
+  InputProps,
+} from "@mui/material";
+import IconButton from "components/IconButton";
+import { SearchIcon } from "components/Icons";
+import TextField from "components/TextField";
+import { useTranslation } from "i18n";
+import { GLOBAL } from "i18n/namespaces";
+import { forwardRef, SyntheticEvent, useState } from "react";
+import { theme } from "theme";
+import { GeocodeResult, useGeocodeQuery } from "utils/hooks";
+
+interface LocationAutocompleteOutlinedProps {
+  className?: string;
+  defaultValue?: string;
+  disableRegions?: boolean;
+  fieldError?: string | undefined;
+  fullWidth?: boolean;
+  id?: string;
+  InputProps?: InputProps;
+  label?: string;
+  name: string;
+  onChange: (value: GeocodeResult | undefined) => void;
+  onClear?: () => void;
+  placeholder?: string;
+  required?: string;
+  showFullDisplayName?: boolean;
+}
+
+const LocationAutocompleteOutlined = forwardRef(function LocationAutocomplete(
+  props: LocationAutocompleteOutlinedProps,
+  ref,
+) {
+  const {
+    className,
+    defaultValue = "",
+    fieldError,
+    fullWidth,
+    id = "location-autocomplete-outlined",
+    InputProps,
+    label,
+    onChange,
+    onClear,
+    placeholder,
+    showFullDisplayName = false,
+  } = props;
+  const { t } = useTranslation([GLOBAL]);
+
+  const [isOpen, setIsOpen] = useState(false);
+  const [value, setValue] = useState<string>(defaultValue);
+
```

Comment:
I made a new one because I wanted it to look different than the existing one. Putting "variant: outlined" on that one wasn't giving me what I wanted.